docs-ref-conceptual/get-started-tutorial-3-use-variables.md

```diff
@@ -0,0 +1,305 @@
+
+Unless the `--output` parameter is specified, these examples rely on a default output configuration of `json` set in [Prepare your environment for the Azure CLI](./get-started-tutorial-1-prepare-environment.md)
+
+### Get JSON dictionary properties of an Azure resource
+
+Using the storage account created in [Learn Azure CLI syntax differences in Bash, PowerShell and Cmd](./get-started-tutorial-2-environment-syntax.md), get the `primaryEndpoints` of your new storage account.
+
+```azurecli-interactive
+az storage account show --resource-group <msdocs-tutorial-rg-00000000> \
+                        --name <msdocssa000000000> \
+                        --query primaryEndpoints
+```
+
+Console JSON dictionary output:
+
+```output
+{
+  "blob": "https://msdocssa00000000.blob.core.windows.net/",
+  "dfs": "https://msdocssa00000000.dfs.core.windows.net/",
+  "file": "https://msdocssa00000000.file.core.windows.net/",
+  "internetEndpoints": null,
+  "microsoftEndpoints": null,
+  "queue": "https://msdocssa00000000.queue.core.windows.net/",
+  "table": "https://msdocssa00000000.table.core.windows.net/",
+  "web": "https://msdocssa00000000.z13.web.core.windows.net/"
+}
+```
+
+### Get individual JSON objects
+
+```azurecli-interactive
+az storage account show --resource-group <msdocs-tutorial-rg-00000000> \
+                        --name <msdocssa000000000> \
+                        --query "[id, primaryLocation, primaryEndpoints.blob, encryption.services.blob.lastEnabledTime]"
+```
+
+Console JSON array (list) output:
+
+```output
+[
+  "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/msdocs-tutorial-rg-00000000/providers/Microsoft.Storage/storageAccounts/   msdocssa000000000",
+  "eastus",
+  "https://msdocssa000000000.blob.core.windows.net/",
+  "yyyy-mm-ddT19:11:56.399484+00:00"
+]
+```
+
+### Rename properties
+
+Rename properties using curly brackets (`{}`) and a comma-delimited list. The new property names cannot contain spaces. This example returns output in `table` format.
+
+```azurecli-interactive
+az storage account show --resource-group <msdocs-tutorial-rg-00000000> \
+                        --name <msdocssa000000000> \
+                        --query "{saName:name, saKind:kind, saMinTLSVersion:minimumTlsVersion}" \
+                        --output table
+```
+
+Console table output.  The first letter of each column is capitalized by design in `--output table`:
+
+```output
+SaName             SaKind     SaMinTLSversion
+-----------------  ---------  -----------------
+msdocssa000000000  StorageV2  TLS1_0
+```
+
+## Filter query results
+
+Combine what you have learned about quoting with what you have just learned about `--query`. These examples apply a filter.
+
+# [Bash](#tab/bash)
+
+```azurecli-interactive
+rgName=<msdocs-tutorial-rg-00000000>
```

Comment:
Maybe it would be helpful to add a note somewhere in the tutorial that spacing is case sensitive? For example I have a habit of adding spaces in between all of my code so when I typed in "rgName = daphneresourcegroup" it gave me an error. Maybe its just me, but just thought I would give a suggestion!

Reply:
The confusing part is that Bash does not allow a space, but PowerShell does. Based on your feedback, I've added this information to the doc.  Thanks for catching this!